atd-vze/src/helpers/map.js

```diff
@@ -9,6 +9,8 @@ export const LOCATION_MAP_CONFIG = {
       type: "raster",
       tiles: [
         `https://api.nearmap.com/tiles/v3/Vert/{z}/{x}/{y}.jpg?apikey=${NEARMAP_KEY}`,
+        // testing basic raster tile server for TX, US
+        //`https://tiles.frankhereford.io/tile/{z}/{x}/{y}.png`,
```

Comment:
Feels a little strange to leave a comment in the code here. Do you think this is useful or better removed?

Reply:
doesn't bother me!